patches/eslint-plugin-react-hooks+4.0.0.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
```

Comment:
What does this patch do?

Reply:
See [discussion](https://github.com/google/web-stories-wp/pull/1589#issuecomment-631450189).